### PR TITLE
Update DP_Bitmasks.mdx

### PR DESCRIPTION
**Pull request by andrewwangva**

### CODE REVIEW

File: content/5_Plat/DP_Bitmasks.mdx [hunk 1]
```diff
@@ -176,7 +176,49 @@ You can often use this to solve subtasks.
 
 ## Solution
 
-<IncompleteSection />
+<Spoiler title =  "Solution">
+
+Let our dp states be defined as $dp[S][i]$ being the amount of routes that visit all the cities in the subset $S$ and end at city $i$. The transitions will then be:
```
Comment (dolphingarlic) [inline content/5_Plat/DP_Bitmasks.mdx:181]:
```suggestion
Let our dp states be defined as $dp[S][i]$ being the number of routes that visit all the cities in the subset $S$ and end at city $i$. The transitions will then be:
```

File: content/5_Plat/DP_Bitmasks.mdx [hunk 2]
```diff
@@ -176,7 +176,49 @@ You can often use this to solve subtasks.
...
+$$
+dp[S][i] = \sum_{x \in adj[i]} dp[S/i][x] \text{ if $x \in S$}
+$$
+where $S/i$ is the subset $S$ without city $i$.
```
Comment (dolphingarlic) [inline content/5_Plat/DP_Bitmasks.mdx:185]: We usually use a backslash (and curly braces for single elements) to denote [set difference](https://mathworld.wolfram.com/SetDifference.html)

**Review by dolphingarlic — CHANGES_REQUESTED** [review]

File: content/5_Plat/DP_Bitmasks.mdx (hunk 2 above)
Comment (bqi343) [reply content/5_Plat/DP_Bitmasks.mdx]: Should use $S \setminus \{i\}$

Comment (bqi343) [reply content/5_Plat/DP_Bitmasks.mdx]: er I forgot the backslashes for the curly brackets:

$S \setminus \{i\}$

File: content/5_Plat/DP_Bitmasks.mdx [hunk 3]
```diff
@@ -176,7 +176,49 @@ You can often use this to solve subtasks.
...
 ## Solution
 
-<IncompleteSection />
+<Spoiler title =  "Solution">
+
+Let our dp states be defined as $dp[S][i]$ being the number of routes that visit all the cities in the subset $S$ and end at city $i$. The transitions will then be:
+$$
+dp[S][i] = \sum_{x \in adj[i]} dp[S \setminus{i}][x] \text{ if $x \in S$}
+$$
+where $S \setminus{i}$ is the subset $S$ without city $i$.
```
Comment (bqi343) [inline content/5_Plat/DP_Bitmasks.mdx:185]: escape the curly braces

File: content/5_Plat/DP_Bitmasks.mdx [hunk 4]
```diff
@@ -176,7 +176,49 @@ You can often use this to solve subtasks.
 
 ## Solution
 
-<IncompleteSection />
+<Spoiler title =  "Solution">
+
+Let our dp states be defined as $dp[S][i]$ being the number of routes that visit all the cities in the subset $S$ and end at city $i$. The transitions will then be:
```
Comment (bqi343) [inline content/5_Plat/DP_Bitmasks.mdx:181]: Why not the following? - Let $dp[S][i]$ be the number of routes that visit all the cities in the subset $S$ and end at city $i$.

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Pull request merged**